open-api/rest-catalog-open-api.yaml

```diff
@@ -846,6 +846,125 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/transactions/commit:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+
+    post:
+      tags:
+        - Catalog API
+      summary: Commit updates to multiple tables in an all-or-nothing operation
+      operationId: commitTransaction
+      requestBody:
+        description:
+          Commit updates to multiple tables in an all-or-nothing operation
+
+
+          A commit for a single table consists of a table identifier with requirements and updates.
+          Requirements are assertions that will be validated before attempting to make and commit changes.
+          For example, `assert-ref-snapshot-id` will check that a named ref's snapshot ID has a certain value.
+
+
+          Updates are changes to make to table metadata. For example, after asserting that the current main ref
+          is at the expected snapshot, a commit may add a new child snapshot and set the ref to the new
+          snapshot id.
+
+
+          A server implementation needs to guarantee that updates across multiple tables are applied in an
+          all-or-nothing operation.
```

Comment:
Not sure you need to say this a second time.

Reply:
good point, I guess we can remove that whole paragraph then